packages/fxa-auth-server/lib/senders/emails/render-to-disk.ts

```diff
@@ -1,21 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
We could make it more prominant in the docs on using `yarn write-emails` to preview the old or new templates. We have this sentence:

> The emails for which the MJML feature flag is enabled can be rendered to disk using the `yarn write-emails` command.

but I think that could be a little clearer 🤔